tests/copilot_helper/test_command_dns.py

```diff
@@ -207,17 +305,61 @@ def test_check_domain_env_flag(check_aws_conn, check_r53, fakefs):
 
     expected = "Checking file: copilot/manifest.yml\nDomains listed in manifest file\n\nEnvironment: dev => Domain: v2.app.dev.test.1234\n\nHere are your Certificate ARNs:\nDomain: v2.app.dev.test.1234\t => Cert ARN: arn:1234\n"
 
+    # Check calls to check_r53
+    # Check calls to check_and_return_aws_session
+
     assert result.output == expected
 
 
+def test_configure_copilot_dir_does_not_exist_exits_with_error(fakefs):
```

Comment:
Took me a moment to grok this, maybe add a word...
```suggestion
def test_configure_when_copilot_dir_does_not_exist_exits_with_error(fakefs):
```